lldb/source/Plugins/TypeSystem/Swift/TypeSystemSwiftTypeRef.h

```diff
@@ -427,6 +427,7 @@ class TypeSystemSwiftTypeRef : public TypeSystemSwift {
 #endif
 
   /// The sibling SwiftASTContext.
+  llvm::Optional<llvm::Triple> m_swift_ast_context_triple = llvm::None;
```

Comment:
I think triple has a default intializer, so we don't really need the Optional wrapper.

Reply:
done